### ISSUE TITLE
[Help]: 1.34.2: setNotifyValue: secondary service not found 'ff22', null, null

### ISSUE THREAD

**Issue by kukafei** [chipweinberger/flutter_blue_plus#1050]
### Requirements

- [X] I've looked at the README 'Common Problems' section

### Have you checked this problem on the example app?

No

### FlutterBluePlus Version

1.34.2

### Flutter Version

3.19.1

### What OS?

Android

### OS Version

android 13

### Bluetooth Module

unkown

### What is your problem?

After upgrading to the latest version, executing 'await characteristic. setNotifyValue (true)' resulted in errors such as' setNotifyValue ',' secondary service not found 'ff22', 'null', and 'null'.  The previous version 1.33.5 did not have this issue.

### Logs

```shell
[38;5;196m[E]PlatformException(setNotifyValue, secondary service not found 'ff22', null, null)
```

**Comment by chipweinberger** [comment]
it would be great if you could figure out the root cause and open a PR.

**Comment by roberto1984** [comment]
Hi,

Same issue here,can not figure out since this morning,I'm using 128 bit uuids

**Comment by chipweinberger** [comment]
this bug was fixed in 1.34.2. please check you are on 1.34.2 or later.

**Comment by chipweinberger** [comment]
try 1.34.4, and create a new issue if you still have problems.
